test/unittest/unittest_blas_kernels_cl.cpp

```diff
@@ -27,17 +27,13 @@
 
 using namespace nntrainer;
 
-static RunLayerContext setUpGpuContext() {
-
+static void setUpGpuContext() {
   auto &ac = nntrainer::ClContext::Global();
-  auto rc = RunLayerContext();
-
-  return rc;
+  ac.initBlasClKernels();
```

Comment:
What about calling `initBlasClKernels()` inside of the `Global()` ?
If not, I think you need to add `cc.initBlasClKernels();` in `layer_node.cpp`  (line 159) as well.

Reply:
There is some issue while calling `initBlasKernels()` from `ClContext` functions itself. `clBuildProgram` was failing. I haven't got a concrete reason for that yet. I assume there is some memory related undefined behaviour when creating a static reference and calling these OpenCL functions within that.

I have kept it such that users can write:
```cpp
auto &cc = nntrainer::ClContext::Global();
cc.initBlasClKernels();
```
This will initialize all the default blas related kernels at the very beginning.
Also as you suggested, it can be added in `layer_node.cpp` as well which will initialize these kernels when any first `LayerNode` is created.